lib/point_quest_web/live/quest_start.ex

```diff
@@ -80,6 +80,7 @@ defmodule PointQuestWeb.QuestStartLive do
       params
       |> StartQuest.new!()
       |> StartQuest.execute()
+      |> dbg()
```

Comment:
**nit:** Can we clean up this dbg, or do we want it in prod logs?

Reply:
whoopsie